cleanup-evicted-pods/deploy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
```

Comment:
Please add an annotation indicating that it's an OOB Job and not part of RKE or Rancher.

Reply:
I added a label for `app: cleanup-evicted-pods` is there another annotation I should add?

Reply:
Thanks, @mattmattox . The label is sufficient.